lib/PINNBench/Burgers/hpo.py

```diff
@@ -10,41 +11,45 @@
 problem.add_hyperparameter((2, 1024), "batch_size", default_value=2)
 problem.add_hyperparameter((200, 300), "epochs", default_value=200)
 problem.add_hyperparameter((0., 5.), "alpha", default_value=.5)
-problem.add_hyperparameter(["relu", "leaky_relu", "tanh", "elu", "gelu", "sigmoid"], "activation", default_value='tanh')
+problem.add_hyperparameter(["relu", "leaky_relu", "tanh", "elu",
+                            "gelu", "sigmoid"],
+                            "activation", default_value='tanh')
 
 
 # define the run function
 def run(config):
-    x, y, val = sanityCheckData()
-    num_layers = config["num_layers"] 
+    nu=0.01/np.pi
+    train, val, test = get_data(NT=200, NX=128, X_U=1, X_L=-1, T_max=1,
+                                Nu=nu)
+    num_layers = config["num_layers"]
     hidden_dim = config["hidden_dim"]
     output_dim = 1
-    nu = 0.01
-    epochs = config["epochs"] 
-    batch_size = config["batch_size"]
+    epochs = config["epochs"]
+    #batch_size = config["batch_size"]
```

Comment:
is the batch size used somewhere?  When you are not using variables you can use default parameter values for k,v dictionary access such as `batch_size = config.get("batch_size", 32)`. This will allow to just changing the `problem` definition.

Reply:
Since we train the PINN using two types of training points (boundary and collocation points), and they have different sizes, I took out the batch size. It is not in the search space anymore. We may add it back later if appropriate.